eagle-topology-check/eagle-topology-app/src/main/java/org/apache/eagle/topology/utils/JMXQueryHelper.java

```diff
@@ -56,7 +56,7 @@
         final Map<String, JMXBean> resultMap = new HashMap<String, JMXBean>();
         final JSONTokener tokener = new JSONTokener(is);
         final JSONObject jsonBeansObject = new JSONObject(tokener);
-        final JSONArray jsonArray = jsonBeansObject.getJSONArray("beans");
+        final JSONArray jsonArray = jsonBeansObject.optJSONArray("beans");
         int size = jsonArray.length();
```

Comment:
if jsonArray is going to be null, can u add that handling as well ? Thanks

Reply:
Agree with @jhsenjaliya,  this change will incur the NPE issue.